### ISSUE TITLE
How can I write a point value as an integer?

### ISSUE THREAD

**Issue by harborhoffer** [node-influx/node-influx#190]
It seems the client sends all integer values as floats. Is there currently any way to specify a value as an integer?

**Comment by connor4312** [comment]
Not right now. What I'd like to have in the future is an optional way to define the Influx "schema" which can be used to smarter type coercion

```js
const client = new Influx({
  // ...
  schema: {
    my_measurement: {
      fields: {
        someString: Influx.STRING,
        someInt: Influx.INTEGER,
        someFloat: Influx.FLOAT,
      },
    },
  },
  // ...
```

**Comment by connor4312** [comment]
With completion of the write API in https://github.com/node-influx/node-influx/commit/0ef0889605cab3d323ff0b4cf4992757a60472d8 I'm closing this issue. Please feel free to take a read over the docs (and/or code) here and let me know if you've suggestions for improvement: [index.ts#L540](https://github.com/node-influx/node-influx/blob/5.0.0/src/index.ts#L540)

**Comment by connor4312** [comment]
These changes have been published in `5.0.0-alpha.1`. You can install this via:

```
npm install --save influx@next
```
